lib/waf.ts

```diff
@@ -25,6 +25,30 @@ export class WebApplicationFirewall extends Construct {
       scope: 'REGIONAL'
     })
 
+    // Implement AWSManagedRulesKnownBadInputsRuleSet
+    const bad_actors_rule_set = {
+      name: 'bad_actors_rule',
+      priority: 0,
+      overrideAction: { none: {} },
+      statement: {
+          managedRuleGroupStatement: {
+              name: 'AWSManagedRulesKnownBadInputsRuleSet',
+              vendorName: 'AWS',
+              excludedRules: [
+                   {name: 'Host_localhost_HEADER'},
+                   {name: 'PROPFIND_METHOD'},
+                   {name: 'ExploitablePaths_URIPATH'}
```

Comment:
Observation: You updated Line 139 to be upper-case as per the docs, but these are not?

Reply:
I copied these directly from the console list of rules in this rule set :shrug:

Reply:
Do the docs recommend capitalising?

Reply:
I was referring to `name` vs `Name` but there seems to be a bit of inconsistency anway.